docs/developer/soffits/README.md

```diff
@@ -0,0 +1,46 @@
+# uPortal Soffits
```

Comment:
For my part, I'm not unwilling to switch the name.

Perhaps just the more generic "Module?"